packages/workflow/src/Interfaces.ts

```diff
@@ -1172,9 +1174,6 @@ export interface INodeType {
 	poll?(this: IPollFunctions): Promise<INodeExecutionData[][] | null>;
 	trigger?(this: ITriggerFunctions): Promise<ITriggerResponse | undefined>;
 	webhook?(this: IWebhookFunctions): Promise<IWebhookResponseData>;
-	hooks?: {
-		[key: string]: (this: IHookFunctions) => Promise<boolean>;
-	};
```

Comment:
Interesting... do you know why we had hooks here? Can't find anything about it.

Reply:
this was never used anywhere, from what I can tell. possibly a relic from a long time ago.